src/libcollections/vec.rs

```diff
@@ -479,18 +479,45 @@ impl<T> Vec<T> {
         }
     }
 
-    /// Shorten a vector to be `len` elements long, dropping excess elements.
+    /// Shortens the vector, keeping the first `len` elements and dropping
+    /// the rest.
     ///
     /// If `len` is greater than the vector's current length, this has no
     /// effect.
     ///
+    /// The [`drain`] method can emulate `truncate`, but causes the excess
+    /// elements to be returned instead of dropped.
+    ///
     /// # Examples
     ///
+    /// Truncating a five element vector to two elements:
```

Comment:
I think "five element" is right. I asked my fiancée and she agreed with me. I wish I knew the exact English rules regarding this, but I don't. Probably something to do with plurality of [attribute nouns](https://en.wikipedia.org/wiki/Noun_adjunct).